components/form/builder/src/index.js

```diff
@@ -0,0 +1,279 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import Select from './components/select'
+import ButtonGroup from './components/button-group'
+import Input from './components/input'
+import TextArea from './components/text-area'
+import AtomButtom from '@schibstedspain/sui-atom-button'
+
+const INITIAL_LIST_ITEM_STATE = []
+const INITIAL_INPUT_ITEM_STATE = ''
+
+const BASE_CLASS = 'sui-FormBuilder'
+const FORM_WRAP_CLASS = `${BASE_CLASS}-wrap`
+const FORM_ITEM_CLASS = `${BASE_CLASS}-item`
+const FORM_ITEM_DISABLED_CLASS = `is-disabled`
+
+class FormBuilder extends Component {
+  constructor({config, submitText}) {
```

Comment:
It would be very interesting to have documentation about the config structure.

Reply:
I'm just create a README with all component info

Reply:
DONE!